scripts/k6/certification/send_code_test.js

```
@@ -0,0 +1,55 @@
+
+export const options = {
+    vus: 200,
+    duration: '1m',
+    thresholds: {
+        'http_req_failed{status:>=500}': ['rate<0.01'],
+        'cert_dup_count': ['count>=1'],
+    },
+};
+
+let certDupCount = new Counter('cert_dup_count');
+let dupSuccessCount = new Counter('cert_dup_success');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'dupSuccessCount' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)